src/components/Reactions/ReactionsListModal.tsx

```diff
@@ -68,7 +68,7 @@ export function ReactionsListModal<
   );
 
   return (
-    <Modal {...modalProps}>
+    <Modal className='str-chat__message-reactions-details-modal' {...modalProps}>
```

Comment:
This could overwrite the default `className`. Perhaps it would be better if we do this instead:
```suggestion
    <Modal {...modalProps} className={clsx('str-chat__message-reactions-details-modal', modalProps.className)} >
```